mixer/tools/codegen/pkg/interfacegen/testdata/apa_tmpl.pb.go

```diff
@@ -1,11 +1,11 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
-// source: bazel-out/local-fastbuild/genfiles/mixer/tools/codegen/pkg/interfacegen/testdata/apa_tmpl.proto
+// source: bazel-out/k8-fastbuild/genfiles/mixer/tools/codegen/pkg/interfacegen/testdata/apa_tmpl.proto
```

Comment:
Unnecessary change?

Reply:
Automated scripts... Will try to remove it, but once Doug has his PR in I assume all those will
be replaced.